src/main/java/com/jaquadro/minecraft/storagedrawers/api/storage/IDrawer.java

```diff
@@ -118,4 +123,17 @@ public interface IDrawer {
     void writeToNBT(NBTTagCompound tag);
 
     void readFromNBT(NBTTagCompound tag);
+
+    /**
+     * Spawns a World-less {@link EntityItem} intended for client-side rendering on Drawer
+     *
+     * @param itemStack The {@link ItemStack} for the {@link EntityItem}
+     * @return The {@link EntityItem} instance (Implementations might cache it)
+     */
+    @SideOnly(Side.CLIENT)
+    default EntityItem getEntityItem(@Nonnull ItemStack itemStack) {
```

Comment:
could you just rename this method to make it explicit that it's used for rendering please